### ISSUE TITLE
Accessibility: Button in VPN settings missing an ARIA label (Windows)

### ISSUE THREAD

**Issue by Murcielago7384** [mullvad/mullvadvpn-app#11120]
### Is it a bug?

- [x] I know this is an issue with the app, and contacting Mullvad support is not relevant.

### I have checked if others have reported this already

- [x] I have checked the issue tracker to see if others have reported similar issues.

### Current Behavior

In the Mullvad VPN Windows desktop app, "> settings > VPN settings", there is a button that is missing an ARIA label that my screen-reader can identify only as "button". It is located in the "custom DNS server" section, and can be found between the two buttons, "add a server" and "more info".

### Expected Behavior

Ideally this button should have a ARIA label as well, so my screen-reader (NVDA) can identify its purpose.

### Steps to Reproduce

1. Once you have a screen-reader running, (NVDA is free and open source, and available from NV Access):
2. Open the Windows Mullvad VPN application and navigate to "settings > VPN settings".
3. You can quickly navigate through the buttons on the page by pressing 'b'. You will hopefully notice that in-between "add a server" and "more info" is one button that is simply announced as "button".

### Failure Logs

```shell

```

### Operating system version

Windows 11 25H2.

### Mullvad VPN app version

Mullvad VPN version 2026.4.0.0

### Additional Information

As mentioned above, NVDA screen-reader may be required to identify this bug. It is free and open source and available from the NV Access website.
Additionally, this was the only unlabelled button that I personally came across, though there may be others that I am not aware of.

**Comment by markus-mullvad** [comment]
Thanks, we will be looking into it!
